Documentation/ExtensionArchitecture/HowTo/ExtendExtbaseModel/Index.rst

```diff
@@ -0,0 +1,208 @@
+.. include:: /Includes.rst.txt
+
+.. _extending-extbase-model:
+
+==========================
+Extending an Extbase model
+==========================
+
+Once you have added an additional field to the database table and
```

Comment:
Maybe add a link to the ext_tables.sql page. Also, with v13 the manual configuration of an additional database field is not necessary anymore.